titiler/application/tests/routes/test_mosaic.py

```diff
@@ -18,7 +18,7 @@
 def mosaic_read_factory(fname: str) -> Callable:
     """Factory method for patching mosaic reading"""
 
-    def _read() -> MosaicJSON:
+    def _read(*args, **kwargs) -> MosaicJSON:  # type: ignore
```

Comment:
I just do `*args: Any, **kwargs: Any` I think Pydantic does that most of the time too
https://github.com/samuelcolvin/pydantic/blob/3f849a368f260caab3288f2ad8fe018d7e903bc3/pydantic/decorator.py#L50-L52